src/main/java/org/hawkular/client/android/activity/MetricDataActivity.java

```diff
@@ -80,7 +82,19 @@ private void setUpToolbar() {
     private void setUpMetricData() {
         showProgress();
 
-        BackendClient.getInstance().getMetricData(getTenant(), getMetric(), this, new MetricDataCallback());
+        BackendClient.of(this).getMetricData(
+            getTenant(), getMetric(), getMetricStartTime(), getMetricFinishTime(), new MetricDataCallback());
+    }
+
+    private Date getMetricStartTime() {
+        Calendar calendar = GregorianCalendar.getInstance();
+        calendar.add(Calendar.MINUTE, -10);
```

Comment:
Was it changed? I remember it was 10 minutes at the UI, maybe I am wrong. I’ll tweak the chart further as a standalone request anyway, the current chart is available to display only 10 minutes without messing it up.